.platform.yaml

```diff
@@ -1,4 +1,4 @@
-required_version: =~ 3
+required_version: 3.2.2.0
```

Comment:
looks like `setup-platform-action` is actually installing `3.2.1.0`, hence the GHA failure

Reply:
so this line should fix it as well, right?

Reply:
It looks like `freckle/setup-platform-action` is installing a version that doesn't have the fix you need.  You need to add the version to `.github/workflows/ci.yml`œ
```
      # Sets up everything needed for platform and stackctl commands to work
      - id: setup
        uses: freckle/setup-platform-action@v7
        with:
          # This is an Org-level secret you can always assume exists
          token: ${{ secrets.FRECKLE_AUTOMATION_GITHUB_TOKEN }}
          version: 3.2.2.0
```

I'm surprised this is necessary.  I'm going to look into why it's not getting the newer version

Reply:
I spoke to this stuff a bit in my other comment. If it's still confusing let's discuss in Slack.

TL;DR: you shouldn't make any changes (IMO), just trigger the deploy and we'll verify through the build logs that it worked. It most likely didn't work the first time because I forgot a step on my end.